src/YourEasyRent/Services/State/IUserStateManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace YourEasyRent.Services.State
+{
+    public interface IUserStateManager
+    {
+        void Started();
```

Comment:
Methods should be VERBs.
Start()
ChooseBrand/SetBrand
ChooseCategory/SetCategory
GoBackTo(...)

Reply:
renamed methods